thunder/tests/test_jit_general.py

```diff
@@ -435,6 +435,19 @@ def foo(a, b):
         assert_close(actual, expected)
 
 
+def test_finfo():
+    def foo(a):
+        return torch.finfo(a.dtype)
+
+    jfoo = thunder.jit(foo)
+    a = torch.randn((2, 2), device="cpu")
+
+    actual = jfoo(a)
+    expected = foo(a)
+
+    assert actual == expected
+
+
```

Comment:
Maybe it would be neat to have both this and the previous check, like

```suggestion
   def bar(a):
        return torch.finfo(a.dtype).min

    jbar = thunder.jit(bar)
    a = torch.randn((2, 2), device="cpu")

    actual = jbar(a)
    expected = bar(a)

    assert_close(actual, expected)


```

Reply:
Yes, me add this right now!